examples/plotting/xy_lims.py

```diff
@@ -14,7 +14,7 @@
 
 import sunpy.data.sample
 import sunpy.map
-
+from sunpy.map import PixelPair
```

Comment:
We can not use this to fix the example, the answer lies in changing other lines of code.